code/src/index.css

```diff
@@ -1,13 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Trispace:wght@400;700&display=swap');
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Trispace', sans-serif;
+  font-size: 13px;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+/* Section for posting a new thought */
+.newThoughtContainer { 
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  margin-top: 20px;
+  width: 350px;
+  height: auto;
+  background-color: rgb(237, 235, 235);
+  border: solid 1.5px  rgba(126, 125, 125, 0.892);
+  box-shadow: 6px 6px;
+}
+
+textarea {
+  font-family: 'Trispace', sans-serif;
+  font-size: 11px;
+  padding: 10px 0px 0px 10px;
+  width: 320px;
+}
+
+.characterCount {
+color:rgb(126, 125, 125);
+font-size: 10px;
+text-align: end;
+margin-top: 0px;
+margin-bottom: 0px;
+}
+
+.sendThought {   /* how do I get the text centered in the button? */
```

Comment:
Try display flex, justify content center on the sendThought p element and remove padding from the heartIcon. Should do the trick :)